crux-lucene/resources/logback-test.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
```

Comment:
don't think we need another one of these files - we can refer to one of the common ones (see `crux-s3`'s `project.clj`)